driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -58,8 +59,7 @@
     private final io.netty.util.Timer scheduler;
 
     private volatile List<Host> triedHosts;
-
-    private volatile Map<InetSocketAddress, Throwable> errors;
```

Comment:
Any significant introduced overhead of creating a CHM for each RequestHandler?  Previously it would only be constructed on error.  I think in any case it would be nice to reduce the initial capacity/load factor/concurrency.